From 9ed8eb61a760bb0708d11946bdc7f38340912ac0 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 8 Sep 2023 14:42:10 +0200
Subject: [PATCH 0521/1141] arm64: dts: st: add power-supply to goodix on
 stm32mp257f-ev1

Add missing fields of touchscreen goodix (power-supply).

Change-Id: Idaa155f763035888a70e066f1862e240e9bc2991
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/326630
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -386,6 +386,8 @@
 		panel = <&panel_dsi>;
 		interrupt-parent = <&gpioi>;
 		interrupts = <13 IRQ_TYPE_EDGE_FALLING>;
+		AVDD28-supply = <&scmi_v3v3>;
+		VDDIO-supply = <&scmi_v3v3>;
 		status = "disabled";
 	};
 };
